src/components/modules/ModuleAttachment.tsx

```diff
@@ -103,14 +103,9 @@ const ModuleAttachmentContent = ({
   return (
     <>
       <View style={styles.innerContent}>
-        <LabelSmall
-          numberOfLines={1}
-          weight="SemiBold"
-          font="ReadexPro"
-          color="blueIO-500"
-        >
+        <LabelSmallAlt numberOfLines={1} color="blueIO-500">
```

Comment:
```suggestion
        <LabelSmallAlt
          numberOfLines={1}
          color={theme["interactiveElem-default"]}
        >
```
Otherwise `blueIO` will be used even when experimental DS is disabled. You should apply the same to `LoadingSpinner`, too:

```
<LoadingSpinner
  testID={activityIndicatorTestId}
  color={theme["interactiveElem-default"]}
/>
```